tutorial03/README.md

```diff
@@ -19,7 +19,7 @@ mkdir my-model-dir
 You have to let ED know where it will be able to find the models. You can do this by setting the ED_MODEL_PATH variable.
 
 ```bash
-export ED_MODEL_PATH=`pwd`/my-model-dir
+source ../setup_env.bash  # This needs to be done from the tutorial specific folder
```

Comment:
This part still does not fully explain what needs to happen to set the model paths. It instead gives the user a magic bash script that does everything for them. 
Write down in the tutorial what paths need to be set. With the sdf models this also includes Gazebo_model_path and gazebo_resource_path